exatomic/core/universe.py

```diff
@@ -303,6 +303,29 @@ def add_molecular_orbitals(self, field_params=None, mocoefs=None,
                                       inplace=inplace, verbose=verbose,
                                       irrep=irrep)
 
+    def to_cube(self, file_name='output', field_number=0):
```

Comment:
This is a bit nitpicky/opinionated. From a user perspective, I think it would be more informative to name this `write_cube` (or similar). I think `to_cube` implies "returns a `Cube` object. That said, I'm approving.